### PR TITLE
Gluon vision transforms

### PR DESCRIPTION
**Pull request by piiswrong**
Add image augmentation utilities to gluon data API.

### CODE REVIEW

**Comment by piiswrong** [comment]
@yzhliu @sxjscience @yajiedesign

**Comment by szha** [comment]
Hi @piiswrong, would you add brief description of what functionality this PR includes?

**Comment by ShownX** [comment]
I am so glad it has been merged.

File: python/mxnet/gluon/data/vision/datasets.py [hunk 1]
```diff
@@ -28,9 +28,9 @@
 import warnings
 import numpy as np
 
-from . import dataset
-from ..utils import download, check_sha1
-from ... import nd, image, recordio
+from .. import dataset
+from ...utils import download, check_sha1
+from .... import nd, image, recordio
```
Comment (szha) [inline python/mxnet/gluon/data/vision/datasets.py]: This move caused the links to existing Vision datasets to break. See https://mxnet.incubator.apache.org/versions/master/api/python/gluon/data.html?highlight=vision#vision

**Pull request merged**